pkg/controller/cassandradatacenter/controller/configmap.go

```diff
@@ -0,0 +1,190 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	cassandraoperatorv1alpha1 "github.com/instaclustr/cassandra-operator/pkg/apis/cassandraoperator/v1alpha1"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"regexp"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"strings"
+)
+
+const (
```

Comment:
I'm still not convinced, sorry. There are numerous other string literals, in various places, that we haven't promoted to constants: the controller name, the container names, the StatefulSet service name, label keys & values, port names, etc.

Why these ones specifically?

Right now, I think I'd prefer that these get re-inlined. We can promote values to constants in the future if and when their value is used in multiple places.

Reply:
I have explained that multiple times, its not for having them exported so they can be used multiple times, _only_, it is also for me having a brief look where are thing _without the necessity to dig in the code_.